doc/ADR/homebrew_tap.md

```diff
@@ -59,13 +91,15 @@
 
 * The binaries are installed without any version tag attached, so they cannot
   live side by side.
-* The install formula doesn't override binaries, so if you already have a CBMC
-  version installed, you need to `remove` it before installing another one:
+* If you install a different version of CBMC and you want to use that one
+  instead, you need to take an extra step after installation to setup the
```

Comment:
Are these instructions for installing multiple cbmc versions side by side / at the same time? Surely if I only wanted to use an alternate cbmc version, I could just uninstall the current version and then reinstall a new version?

Reply:
I believe the comment in the question below answers this question.

If not, let me know and I can clarify.